client/pages/details.jsx

```diff
@@ -68,6 +80,22 @@ export default class ParkDetails extends React.Component {
       return;
     }
     const park = this.state.results;
+    let rating;
+    if (this.state.parkRating.rating === null) {
+      rating = 'N/A';
+    } else {
+      const score = Number(this.state.parkRating.rating);
+      Math.round(score);
```

Comment:
This line 88 isn't really doing anything because numbers are immutable. You'll want to capture `.round()`'s return value